src/components/ScholarshipDetailCard.jsx

```diff
@@ -1,29 +1,129 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button, Box, Link, makeStyles, Typography } from '@material-ui/core';
+import {
+  Button,
+  Box,
+  Link,
+  makeStyles,
+  Typography,
+  Chip,
+  Grid,
+  Divider,
+} from '@material-ui/core';
+import { Share, Send, Info } from '@material-ui/icons';
 import ScholarshipAmount from '../types/ScholarshipAmount';
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme) => ({
+  actionsWrapper: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(4),
```

Comment:
I see in a couple of other CSS classes but can we make it such that, in general, top margins match bottom margins? Or at least I'd like to see some consistency between the various sections.